app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
Storing `self` in the class attribute `people` can lead to unexpected behavior, especially if multiple `Person` instances are created. Consider revising this approach to avoid potential issues with shared state across instances.